common/src/helpers/router.tsx

```diff
@@ -73,42 +73,15 @@ export const useRouter = (
   return component;
 };
 
-export const Link: React.FC<{ href: string; title: string }> = ({
-  href,
-  title,
-  children,
-  ...props
-}): React.ReactElement => (
-  <a
-    href={href}
-    title={title}
-    {...props}
-    onClick={(event): void => {
-      event.preventDefault();
-      navigate(href);
-    }}
-  >
-    {children}
-  </a>
-);
-
 /* eslint-disable no-useless-escape */
 function stripTrailingSlash(str: string): string {
```

Comment:
`stripTrailingSlash` is declared but its value is never read. Is there a plan to use this function in the future?

Reply:
Uh I guess not? We can probably remove it along with baseUrl()